### PR TITLE
Support flow export notification

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: src/list/NotificationList.ts [hunk 1]
```diff
@@ -100,6 +100,9 @@ export class NotificationList extends TembaList {
         } else if (notification.export.type === 'ticket') {
           icon = Icon.tickets_export;
           body = 'Exported tickets';
+        } else if (notification.export.type === 'flows') {
```
Comment (rowanseymour) [inline src/list/NotificationList.ts:103]: how about `definition` since they're not just flows? 

`contact`, `message`, `results` and `definition`

Comment (norkans7) [reply src/list/NotificationList.ts:103]: Okay

Comment (norkans7) [reply src/list/NotificationList.ts:103]: And the body can be `Exported definitions` ? @rowanseymour

Comment (rowanseymour) [reply src/list/NotificationList.ts]: Sure

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**